tasks/gulp/copy-to-destination.mjs

```diff
@@ -170,7 +170,7 @@ async function generateMacroOptions (file) {
  * Parse YAML file content as JavaScript
  *
  * @param {import('vinyl')} file - Component data ${componentName}.yaml
- * @returns {Promise<{ examples?: unknown[]; params?: unknown[] }>} Component options
+ * @returns {Promise<{ examples?: Record<string, unknown>[]; params?: Record<string, unknown>[] }>} Component options
```

Comment:
**question** Non blocking, but what does the `Record` type represent?

Reply:
It's a [utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) for objects that are always keyed with the same name/value pair

Reply:
Oh didn't think to look in the built-in types, cheers 😄